### ISSUE TITLE
Supporting ConTeXT in rmarkdown

### ISSUE THREAD

**Issue by jooyoungseo** [rstudio/rmarkdown#1713]
Would there be any way for us to support `ConTeXT` in `rmarkdown`?

I have found that `--pdf-engine` flag does not work with current `rmarkdown`.

According to [this thread discussion](https://github.com/jgm/pandoc/issues/5409) a tagged pdf file can be made by `ConTeXT` so I was wondering if there would be any way for me to produce `Rmd` through `ConTeXT` engine. Of course, users have to install `ConTeXT` to use that feature.

I have tested with the following demo `Rmd` file:

````
---
output:
  pdf_document:
    pandoc_args: --pdf-engine=context
---

# Introduction

Hello, world!

# Body

This is body!

# Conclusion

Good bye!
````

**Comment by RLesur** [comment]
I think a specific output format is required. I fully understand the motivation for PDF/A, so I'll give it a try.

**Comment by atusy** [comment]
My PR #1715 extends `md_document` and supports ConTeXt.
This can be an alternative of the future specific output by @RLesur .

**Comment by yihui** [comment]
I just merged #1725. Please test the development version:

```r
remotes::install_github('remotes/rmarkdown')
```

Special thanks to @RLesur for the PR!

**Comment by jooyoungseo** [comment]
Thank you very much!
I have just tested and confirmed that everything is working beautifully!!
